app/Project/ProjectProvider.php

```diff
@@ -30,12 +30,8 @@ public function register(Container $pimple)
             return new ProjectConverter($pimple['project_repository']);
         };
 
-        $pimple['default_consumer_group'] = function (Container $pimple) {
-            return $pimple['config']['default_consumer_group'];
-        };
-
-        $pimple['uitpas_permission_group'] = function (Container $pimple) {
-            return $pimple['config']['uitpas_permission_group'];
+        $pimple['permission_groups'] = function (Container $pimple) {
+            return $pimple['config']['permission_groups'];
```

Comment:
Whoops @brampauwelyn , you merged this without an explicit review approval and now I notice this service isn't necessary anymore I think? 😄 Can you remove it in a follow-up PR please? Thanks! 👌

Reply:
Indeed whoops. 
I made a follow up PR: #199